erpnext/crm/doctype/opportunity/opportunity.json

```diff
@@ -409,6 +411,20 @@
    "fieldtype": "Link",
    "label": "Converted By",
    "options": "User"
+  },
+  {
+   "default": "0",
```

Comment:
Save this doctype again, the `modified` time doesn't seem to have changed. That's what Frappe uses for syncing new fields.